tests/test_binary_sensor.py

```
@@ -249,3 +249,52 @@ async def test_binary_sensor_none_checks(hass: HomeAssistant, hass_client) -> No
+        "name": "Gear in Park",
+        "type": "can_state",
+        "can_id": "0x2C0",
+        "match_payload": "* * 00 * * * * *",
+    }
+    entity = WiCANCanConditionBinarySensorEntity(mock_config_entry, cond_def)
+    entity.hass = hass
+    entity.entity_id = "binary_sensor.wican_device_cando_gear_park"
+
+    # Simulate state update matching
+    coordinator.handle_webhook_data({"can_states": {"0x2C0": {"data": "0002002000000000"}}})
+    entity._handle_coordinator_update()
```

Ruff (0.16.3)

[error] Private member accessed: `_handle_coordinator_update`

(SLF001)